core/common/src/main/java/alluxio/Seekable.java

```diff
@@ -26,4 +26,9 @@
    * @param pos the position to seek to, it must be between 0 and the end of the stream - 1
    */
   void seek(long pos) throws IOException;
+
+  /**
+   * @return the current position of the stream
+   */
+  long getPos();
```

Comment:
What are your thoughts on creating a separate `Positioned` interface instead of adding the method here? That way we have flexibility to have `Seekable`s which don't expose current position.